experience.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="experience.css" />
+    <title>Experience Page</title>
+  </head>
+  <body>
+    <nav>
+      <a href="home.html">Home</a>
+      <a href="bio.html">Bio</a>
+      <a href="experience.html">Experience</a>
+      <a href="portfolio.html">Portfolio</a>
+    </nav>
+    <main>
+      <section class="education">
+        <h1>Education</h1>
+
+        <ul class="education-list">
```

Comment:
I appreciate the thoughtful class names that you've used on this page, as well as other HTML files!